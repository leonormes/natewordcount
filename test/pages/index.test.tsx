import React from 'react'
import { render, fireEvent } from '../testUtils'
import Home from '../../pages/index'

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Home />, {})
    expect(asFragment()).toMatchSnapshot()
  })

  it('clicking button triggers alert', () => {
    const { getByText } = render(<Home />, {})
    fireEvent.click(getByText('Count'))
  })
})
