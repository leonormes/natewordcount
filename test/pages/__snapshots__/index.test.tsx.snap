// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="jsx-1276654382 container"
  >
    <main
      class="jsx-1276654382"
    >
      <h1
        class="jsx-1276654382 title"
      >
        URL Word Counter
      </h1>
      <button
        class="jsx-1276654382"
      >
        Test Button
      </button>
    </main>
  </div>
</DocumentFragment>
`;
